### PR TITLE
Extending test cases.

### PR DESCRIPTION
**Pull request by walter-weinmann**
Additional test cases from checking the railroad diagrams. 

Test cases which are not accepted yet by the parser start with the characters "`% ? `" since I could not decide if the change should be in the syntax definition or in the parser.

### CODE REVIEW

**Comment by c-bik** [comment]
@walter-weinmann Can you please resolve this conflict? I can then merge and work on the new test cases, every-now-and-then.

**Comment by walter-weinmann** [comment]
Sorry - I can't see any conflicts - maybe the following old lines in select.tst are causing the trouble:

```
<<"select a < 0, f(a+2 as d), f2(a < 3 as f) from abc">>.

<<"select 'öüäéèà', 'شلاؤيثبلتهتنمةىخ','นี่คือการทดสอบ' from dual"/utf8>>.

<<"sElect
...
```

**Comment by c-bik** [comment]
image

```sh
$ git checkout -b walter-weinmann-sqlparse_wwe_ext_test_cases master
...
$ git pull https://github.com/walter-weinmann/sqlparse.git sqlparse_wwe_ext_test_cases
From https://github.com/walter-weinmann/sqlparse
 * branch            sqlparse_wwe_ext_test_cases -> FETCH_HEAD
Auto-merging test/cursor.tst
CONFLICT (add/add): Merge conflict in test/cursor.tst
Automatic merge failed; fix conflicts and then commit the result.
```

Conflicted `test/cursor.tst`:
```erlang
%%-*- mode: erlang -*-
%%-*- coding: utf-8 -*-

% Test control options
[{verbose, 0}, {tests, []}].

%% 
<<<<<<< HEAD
%% TESTS
%%

"CLOSE name_cursor_1".
"DECLARE name_cursor_1 CURSOR FOR SELECT * FROM name_table_1".
"FETCH name_cursor_1 INTO name_column_1".
"OPEN name_cursor_1".
"DELETE FROM table_name WHERE CURRENT OF name_cursor".
"UPDATE name_table SET name_column_1 = :value_1 WHERE CURRENT OF name_cursor".
"UPDATE name_table SET name_column_1 = :value_1, name_column_2 = :value_2 WHERE CURRENT OF name_cursor".
=======
%% TESTS CLOSE
%%

% ? "CLOSE name_cursor_1".

%%
%% TESTS DECLARE
%%

% ? "DECLARE name_cursor_1 CURSOR FOR SELECT * FROM name_table_1".


%%
%% TESTS DELETE
%%

% ? "DELETE FROM table_name WHERE CURRENT OF name_cursor".

%%
%% TESTS FETCH
%%

% ? "FETCH name_cursor_1 INTO name_column_1".

%%
%% TESTS OPEN
%%

% ? "OPEN name_cursor_1".

%%
%% TESTS UPDATE
%%

% ? "UPDATE name_table SET name_column_1 = :value_1 WHERE CURRENT OF name_cursor".
% ? "UPDATE name_table SET name_column_1 = :value_1, name_column_2 = :value_2 WHERE CURRENT OF name_cursor".


>>>>>>> 059ed285bba469e2e8a16225710ce804add8bfbc
```

Hope it helps.

**Comment by walter-weinmann** [comment]
Thanks for the help - should be ok now.

File: test/cursor.tst [hunk 1]
```diff
@@ -15,3 +15,4 @@
 "DELETE FROM table_name WHERE CURRENT OF name_cursor".
 "UPDATE name_table SET name_column_1 = :value_1 WHERE CURRENT OF name_cursor".
 "UPDATE name_table SET name_column_1 = :value_1, name_column_2 = :value_2 WHERE CURRENT OF name_cursor".
+>>>>>>> origin/master
```
Comment (c-bik) [inline test/cursor.tst]: Please remove this git diff line

**Comment by walter-weinmann** [comment]
Sorry - done.

**Comment by c-bik** [comment]
Thanks @walter-weinmann

**Pull request merged**